index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- fontawesomecdn -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <!-- link to css file -->
+    <link rel="stylesheet" href="./style.css">
+    <title>Lynette || My Portfolio</title>
+</head>
+<body>
+    <header>
+
+        
+        <div class="container">
+       <div class="wrapper">
+                <nav>
+                    <a href="#" class="welcome">Welcome</a>
```

Comment:
- [ ] The previous reviewer did provide you with a structural guide. The logo should be out of the `nav` tag.